mesaflash.c

```diff
@@ -193,9 +194,9 @@ void print_usage() {
     printf("  --rpo             Read hostmot2 variable directly at 'address'.\n");
     printf("  --wpo             Write hostmot2 variable directly at 'address'\n");
     printf("                    with 'value'.\n");
-    printf("  --set             Set board IP address in EEPROM to n.n.n.n (only\n");
+    printf("  --set             Set board IP address in EEPROM memory to n.n.n.n (only\n");
```

Comment:
Should this (and one other reference above) be changed to FLASH memory like the other references, or is there a separate EEPROM for the board IP address?

Reply:
No, it really EEPROM memory chip(93C46) used for storage IP, MAC and board name.
FPGA configuration stored in FLASH memory chip.

Reply:
thanks, that is helpful clarification for me.